From b0f3cadb49ed0dda48f4a99f31043bacc2c6f9fb Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Thu, 27 Jul 2023 09:46:17 +0200
Subject: [PATCH] Force debugging

---
 src/cms_common.h | 2 ++
 src/daemon.h     | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/src/cms_common.h b/src/cms_common.h
index 35a128a..ac98619 100644
--- a/src/cms_common.h
+++ b/src/cms_common.h
@@ -25,6 +25,8 @@
 #include "util.h"
 #include "password.h"
 
+#define PESIGN_DEBUG 1
+
 #define save_port_err() \
 	for (error_t saved_errno_0_ = 0, saved_errno_1_ = PORT_GetError(); saved_errno_0_ < 1; saved_errno_0_++, PORT_SetError(saved_errno_1_))
 
diff --git a/src/daemon.h b/src/daemon.h
index 72470a7..b80424e 100644
--- a/src/daemon.h
+++ b/src/daemon.h
@@ -7,6 +7,8 @@
 #ifndef DAEMON_H
 #define DAEMON_H 1
 
+#define PESIGN_DEBUG 1
+
 extern int daemonize(cms_context *ctx, char *certdir, int do_fork);
 
 typedef struct {
-- 
2.41.0

